config/initializers/workless.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.config.after_initialize do
+  Delayed::Job.scaler = :null if ENV['WORKLESS_DISABLED'] == 'true'
```

Comment:
Can you also update the deployment.yaml and worker.yaml so that they read the value of `WORKLESS_DISABLED` from the secrets bag?